my-webxr-app/jest.config.js

```diff
@@ -0,0 +1,9 @@
+// jest.config.ts
+/** @type {import('jest').Config} */
+const config = {
```

Comment:
have better var naming style jest config

Reply:
This change has been completed